.gon.arm64.hcl

```diff
@@ -0,0 +1,15 @@
+source = ["./dist/macos-arm64_darwin_arm64/L2-node"]
```

Comment:
the arm64 signing isn't supported by gon yet :( Shall I clean up this file for now?